### PR TITLE
[Improve][Connector-V2][Doris] Change Doris Config Prefix

### PR DESCRIPTION
**Pull request by TaoZex**
## Purpose of this pull request
reference https://github.com/apache/incubator-seatunnel/pull/3719
Change Doris Config Prefix

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)

### CODE REVIEW

**Comment by TaoZex** [comment]
@Hisoka-X PTAL

File: docs/en/connector-v2/sink/Doris.md [hunk 1]
```diff
@@ -75,11 +75,9 @@ Using as a multiplier for generating the next delay for backoff
 
 The amount of time to wait before attempting to retry a request to `Doris`
 
-### sink.properties.*  [doris stream load config]
+### doris.config [map]
```
Comment (EricJoy2048) [inline docs/en/connector-v2/sink/Doris.md:78]: Can you add more instructions about this option? For example where can users find these parameters?

Comment (TaoZex) [reply docs/en/connector-v2/sink/Doris.md:78]: Thanks，I added the stream load official doc link.

**Comment by Hisoka-X** [comment]
Hi, put this change to doris e2e module, thanks.

**Comment by TaoZex** [comment]
> Hi, put this change to doris e2e module, thanks.

ok，I have fixed it.

**Review by Hisoka-X — APPROVED** [review]

**Review by hailin0 — APPROVED** [review]

**Pull request merged**